build-docker.sh

```diff
@@ -320,49 +303,51 @@ done
 
 # build legacy
 
-for BITCOIN_ONLY in ${VARIANTS_legacy[@]}; do
+if echo "${MODELS[@]}" | grep T1B1; then
```

Comment:
do not print the match
```suggestion
if echo "${MODELS[@]}" | grep -q T1B1 ; then
```